luxtronik/__main__.py

```diff
@@ -47,3 +47,6 @@ def main() -> int:
     sys.argv.pop(1)
     # call the corresponding command
     commands[args.command]()
+
+if __name__ == "__main__":
+    main()
```

Comment:
Obviously this is a trivial change, still wondering: Is this actually related to "improvements for windows"?